### PR TITLE
Fix ActivePage declarations

### PR DESCRIPTION
**Pull request by tstivers1990**
* Some of the `ViewData["ActivePage"]` declarations were missing.
* `DeletePersonalData` and `DownloadPersonalData` should set `ActivePage` to `PersonalData` as there is no navigation element for `DeletePersonalData` or `DownloadPersonalData`.
* Remove `DeletePersonalData` and `DownloadPersonalData` variables and methods as they're no longer used by anthing.
* Make `PageNavClass` private as it is never used outside of its class.

### CODE REVIEW

**Comment by tstivers1990** [comment]
The `TwoFactorAuthentication` page was also missing the `ActivePage` assignment. I've added that in the latest commit.

**Comment by HaoK** [comment]
@javiercn do these changes look reasonable to you?

**Comment by javiercn** [comment]
Thanks for contacting us @tstivers1990 
Could you please file or point the PR to an issue describing the problem and what the proposed changes do (What's the visual before/after)

**Comment by tstivers1990** [comment]
See #1866

File: src/UI/Areas/Identity/Pages/Account/Manage/ManageNavPages.cs [hunk 1]
```diff
@@ -12,10 +12,6 @@ public static class ManageNavPages
 
         public static string ChangePassword => "ChangePassword";
 
-        public static string DownloadPersonalData => "DownloadPersonalData";
```
Comment (javiercn) [inline src/UI/Areas/Identity/Pages/Account/Manage/ManageNavPages.cs]: The changes look good. The only thing  we need to change are the removed members and the visibility changes. Is there any issue if you remove those changes? Those are breaking changes and we would ideally like to take this into one of our patch releases, so we need to revert those changes even if the methods are not used.

Comment (tstivers1990) [reply src/UI/Areas/Identity/Pages/Account/Manage/ManageNavPages.cs]: @javiercn Yeah, my thought process at the time was that these changes shouldn't break anything, as it's scaffolded code. But now that you mention it, if someone were to have scaffolded one of the pages that was changed, and then updates Identity UI with these changes, it's likely to break something.

I'm not sure how you want me to go about this. The breaking changes are in two commits, so they can easily be reverted. Should the breaking changes be reverted and put into a separate pull request?

Comment (javiercn) [reply src/UI/Areas/Identity/Pages/Account/Manage/ManageNavPages.cs]: @tstivers1990 I think scraping them all together is fine. While it's not super clean, its in a .Internal namespace so it doesn't really matter. I don't want to trouble you into sending another PR as there's not much value on fixing it. Scraping the commits with the breaking changes will work fine.

Thanks for your contribution

**Comment by tstivers1990** [comment]
The breaking changes should be removed.

**Comment by javiercn** [comment]
Merged! Thanks for your contribution!

**Pull request merged**